src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -171,6 +171,12 @@ public static void AddTokenizers(this IServiceCollection services)
                 WebAuthnCredentialCreateOptionsTokenable.DataProtectorPurpose,
                 serviceProvider.GetDataProtectionProvider(),
                 serviceProvider.GetRequiredService<ILogger<DataProtectorTokenFactory<WebAuthnCredentialCreateOptionsTokenable>>>()));
+        services.AddSingleton<IDataProtectorTokenFactory<SsoEmail2faSessionTokenable>>(serviceProvider =>
+            new DataProtectorTokenFactory<SsoEmail2faSessionTokenable>(
+                SsoEmail2faSessionTokenable.ClearTextPrefix,
+                SsoEmail2faSessionTokenable.DataProtectorPurpose,
+                serviceProvider.GetDataProtectionProvider(),
+                serviceProvider.GetRequiredService<ILogger<DataProtectorTokenFactory<SsoEmail2faSessionTokenable>>>()));
```

Comment:
Did this get added from a merge or something?

Reply:
Ah yeah, I did a merge from master into my client branch and must've done one on the server branch out of habit. If you can merge `master` into `fido2` then I'll merge `fido2` into this one. Either that or I rebase and remove that merge, but then I'll have to force push which would be nice to avoid